Runtime/Model/JsonData/Annotations.cs

```diff
@@ -19,28 +19,18 @@ public class Annotations
         /// </summary>
         internal static Dictionary<string, string> _environmentVariablesCache;
 
-        /// <summary>
-        /// Determinate if static helper should load environment variables or not.
-        /// </summary>
-        internal static bool VariablesLoaded;
-
         /// <summary>
         /// Loaded environment variables
         /// </summary>
         public static Dictionary<string, string> EnvironmentVariablesCache
         {
             get
             {
-                if (VariablesLoaded == false)
-                {
-                    _environmentVariablesCache = SetEnvironmentVariables();
-                    VariablesLoaded = true;
-                }
                 return _environmentVariablesCache;
             }
             set
             {
-                _environmentVariablesCache = value;
+                _environmentVariablesCache = SetEnvironmentVariables(_environmentVariablesCache);
```

Comment:
Did you mean 
```suggestion
                _environmentVariablesCache = SetEnvironmentVariables(value);
```
?

Reply:
You're correct. I cherry-picked not enough changes. thanks!